### PR TITLE
fix(ohlc): update to use orderbook vals if trades <= 10

### PR DESCRIPTION
**Pull request by moo-onthelawn**
context: https://dydx-team.slack.com/archives/C07DRA7TZS7/p1724173168660419?thread_ts=1723654679.130969&cid=C07DRA7TZS7

### CODE REVIEW

File: src/lib/tradingView/utils.ts [hunk 1]
```diff
@@ -29,7 +31,7 @@ const getOhlcValues = ({
 }) => {
   const useOrderbookCandles =
     orderbookCandlesToggleOn &&
-    trades === 0 &&
+    trades <= MIN_NUM_TRADES_FOR_ORDERBOOK_PRICES &&
```
Comment (tyleroooo) [inline src/lib/tradingView/utils.ts:34]: maybe rename to numTrades or tradesCount since trades kinda makes me think it's a list.

**Review by tyleroooo — APPROVED** [review]

File: src/lib/tradingView/utils.ts [hunk 2]
```diff
@@ -8,6 +8,8 @@ import { Themes } from '@/styles/themes';
 
 import { AppTheme, type AppColorMode } from '@/state/configs';
 
+const MIN_NUM_TRADES_FOR_ORDERBOOK_PRICES = 10;
```
Comment (yogurtandjam) [inline src/lib/tradingView/utils.ts:11]: [non-blocking] this function is a great opportunity for a small unit test 😄

**Review by yogurtandjam — APPROVED** [review]

File: src/lib/tradingView/utils.ts (hunk 2 above)
Comment (tyleroooo) [reply src/lib/tradingView/utils.ts:11]: 🤮

File: src/lib/tradingView/utils.ts (hunk 1 above)
Comment (moo-onthelawn) [reply src/lib/tradingView/utils.ts:34]: ah yeah it's curr trades everywhere so will [follow uppp](https://linear.app/dydx/issue/CT-1138/rename-trades-numtrades) - will get ur unit test in there @yogurtandjam

**Pull request merged**